estate/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+{
+    'name': "Real Estate",
+    'version': '1.0',
+    'summary'  "Manage properties and buyers."
```

Comment:
```suggestion
    'summary':  "Manage properties and buyers."
```

there is a missing colon here :D